src/App.jsx

```diff
@@ -14,70 +16,68 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [isActive, selectedGood] = useState('Jam');
```

Comment:
we usually name state like [state, setState]
```suggestion
  const [selectedGood, setSelectedGood] = useState('Jam');
```